### PR TITLE
61 land cover avg

### PR DESCRIPTION
**Pull request by cstillwellusgs**
Lines 256-261 of '1_fetch/src/get_sb_data.r' were used to identify 10 COMIDs with incomplete land cover data (the sum of land cover classes fell short of 100%). After further investigation, all 10 of these COMIDs are associated with streamgages in the Gages2.1 dataset with upstream area from Canada. Please advise on how to handle these types of basins.

### CODE REVIEW

File: 1_fetch/src/get_sb_data.R [hunk 1]
```diff
@@ -121,7 +121,12 @@ reduce_sb_table <- function(sb_table_full, sb_var_list, outdir) {
   #additional IDs to get missing values
   #if there are missing values, edit this function with the IDs you want to add
   #or, edit the sb_var_list with the IDs you want.
-  retain_ids <- c()
+  retain_ids <- c("5734acafe4b0dae0d5de622d", 
+                  "5730f062e4b0dae0d5db1fbe", 
+                  "573362bce4b0dae0d5dd6193", 
+                  "574ddc8ce4b07e28b66901b3",
+                  "574f3e86e4b0ee97d51abf31",
+                  "574f238fe4b0ee97d51a8916")
```
Comment (jds485) [inline 1_fetch/src/get_sb_data.R:129]: I edited the xslx table to include these
```suggestion
  retain_ids <- c()
```

**Comment by jds485** [comment]
> the sum of land cover classes fell short of 100%

How close to 100% are they? We could assume the proportion holds and rescale to get the sum to 100%

**Comment by cstillwellusgs** [comment]
> How close to 100% are they? We could assume the proportion holds and rescale to get the sum to 100%

Varies. Highest is 97%, lowest is 4%. Just depends on how much is flowing in from Canada.

**Comment by jds485** [comment]
Okay, 4% would be too small to rescale. Do you know if these sites also have gaps for other datasets? It seems likely they would for soil properties, for example. Might be best to drop these sites. Would be interested in your thoughts for how attributes are assigned for catchments that overlap Canada, @ajsekell

**Comment by ajsekell** [comment]
I haven't had to deal with an issue like this but we simply won't be able to get complete datasets for the contributing areas within Canada.

We could exclude sites with contributing area less than 90% (or some number) in USA.

Andrew Sekellick
Physical Scientist
USGS MD-DE-DC Water Science Center
443-498-5580
***@***.******@***.***>

________________________________
From: Jared D. Smith ***@***.***>
Sent: Wednesday, April 27, 2022 9:43 AM
To: USGS-R/regional-hydrologic-forcings-ml ***@***.***>
Cc: Sekellick, Andrew J ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [USGS-R/regional-hydrologic-forcings-ml] 61 land cover avg (PR #99)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



Okay, 4% would be too small to rescale. Do you know if these sites also have gaps for other datasets? It seems likely they would for soil properties, for example. Might be best to drop these sites. Would be interested in your thoughts for how attributes are assigned for catchments that overlap Canada, @ajsekell<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fajsekell&data=05%7C01%7Cajsekell%40usgs.gov%7Cfe54b35d99d348c387db08da285462e8%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637866640142778500%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=qbRBQqiztFYKwww%2BGo7l0811o0ICCX6OvXrPEW8h6XY%3D&reserved=0>

**Comment by cstillwellusgs** [comment]
Sounds good to me, I will filter out the COMIDs that are less than 90% in CONUS. I will handle this in the new issue I just created #101.

**Comment by jds485** [comment]
@cstillwellusgs: are we closing this issue? I think #103 PR addresses these changes

**Comment by cstillwellusgs** [comment]
Yes this issue can be closed and this PR deleted, all handled in #101 and #103

**Pull request closed without merging**